selfdrive/crash.py

```diff
@@ -1,67 +1,21 @@
 """Install exception handler for process crash."""
-import os
```

Comment:
Revert crash.py

Reply:
I'll close this pull request and pushes another request only change longitudinal_planner.py.